packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -1,3 +1,20 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-navigation-logo-active-border-color: Specifies the border color of the component when active property is true.
+ * @prop --calcite-navigation-logo-active-text-color: Specifies the text color of component when active property is true.
+ * @prop --calcite-navigation-logo-background-color-active: Specifies the background color of the component when active.
+ * @prop --calcite-navigation-logo-background-color-hover: Specifies the background color of the component when hovered.
+ * @prop --calcite-navigation-logo-border-color-active: Specifies the border color of the component when active.
+ * @prop --calcite-navigation-logo-description-text-color: Specifies text color of description of component.
+ * @prop --calcite-navigation-logo-heading-text-color: Specifies the text color of heading of component.
+ * @prop --calcite-navigation-logo-text-color-active: Specifies the text color of component when active.
```

Comment:
Seems like we’d want hover mapped for text as well with the new stateful pattern.

But; it does beg the question - do we need stateful tokens for the active state? It’s reasonable to imagine a theming use case that would want the ability to adjust this state on hover. Of course a user can target this state in their css but it breaks the new pattern we were told to use.

Reply:
Good catch, don't think `--calcite-navigation-logo-text-color-active` is used in active state of the component. Removed this and updated the components to follow the pattern of reassigning tokens for each state. For this component we can leverage that since all the state styles are applied at root element in shadowDOM rather than subcomponents or children elements. Thoughts?

Reply:
Thought the pattern of using stateful vs stateless tokens is dependent on the component structure. navigation-logo is not being used as subcomponent in shadowDOM of any other component  and user can theme the state in css. Do we want to avoid that

Reply:
Yeah, I'm not sure honestly. I thought the stateless tokens were only for components inaccessible to a dev-user, and others were set by a user with custom-element:hover, etc. cc @alisonailea for clarification.

Reply:
I don't think we need stateful tokens here.